JapaneseFilter/sections/specific.txt

```diff
@@ -44,6 +44,7 @@ tokyo-sports.co.jp,soccerdigestweb.com,wiki.yjsnpi.nu,athleterecipe.com,javkimoc
 !
 !
 ||episodesex.org/banner/
+cookpad.com##div > div[data-ad-slot^="spweb-"]
```

Comment:
I don't see it on desktop, so you can move it to AdGuard Mobile Ads as
```
cookpad.com##div[data-ad-slot^="spweb-"]
```

Reply:
Sorry, I mistook that.